### ISSUE TITLE
Live track Interrupted: don't draw line

### ISSUE THREAD

**Issue by pyrog** [skylines-project/skylines#541]
When there is no data, don't draw a line between the "gap".

In this example, the straith  line correspond to the travel of the pilote between the landing and takeoff places, GPS probably off 😉 

<img width="702" alt="capture d ecran 2016-11-02 a 16 39 58" src="https://cloud.githubusercontent.com/assets/2158081/19935750/6ee1345e-a11b-11e6-898b-83d5f1e5e0e6.png">

**Comment by Turbo87** [comment]
This is unfortunately not that easy. We currently only draw a single line string, but if we implement your suggestion we would first need to run analysis on where the gaps are and then split into multiple line strings. 

also I'd prefer it if the gaps were still connected, but with a dotted line.

**Comment by pyrog** [comment]
The pilot confirm that he as correctly stop XCSoar between the 2 flights.
So this is a bug in this app or a protocol issue (how to say that the tacking is finished)

> I'd prefer it if the gaps were still connected, but with a dotted line.

👍 Yes for missing data (loss of radio coverage during the flight)

**Comment by Turbo87** [comment]
> It's quite easy: if we receive no data during 2 or 3 tracking period, so the last "data" is null.

and what is a "tracking period"? keep in mind that the tracking interval is configurable in XCSoar.

> The pilot confirm that he as correctly stop XCSoar between the 2 flights.
> So this is a bug in this app or a protocol issue.

The protocol wasn't designed that way. It simply tells our server the location, but it does not have a session or anything like that.

**Comment by pyrog** [comment]
> and what is a "tracking period"? keep in mind that the tracking interval is configurable in XCSoar.

Yes, but XCSoar know this parameter. It could send it to SkyLines

> The protocol wasn't designed that way.

Ok. So the protocol could evolve with having a new parameter/command that say "start/end of track".
It could be simply send position with null lat/lon (I don't know yet the protocol)
